draft-ietf-quic-transport.md

```diff
@@ -646,10 +657,10 @@ that the client selected.
 
 When the client receives a Version Negotiation packet from the server, it should
 select an acceptable protocol version.  If the server lists an acceptable
-version, the client selects that version and resends all packets using that
-version. The resent packets MUST use new packet numbers.  These packets MUST
-continue to have the VERSION flag set and MUST include the new negotiated
-protocol version.
+version, the client selects that version and reattempts to created a connection
+using that version.  The client MUST update packet numbers when attempting to
```

Comment:
I'm not sure what "client MUST update packet numbers" means?  

I would change "reattempts to created a connection using that version." to "attempts to create a new connection using that version." and then remove this sentence, because I think it's clear a new connection has a new packet number space.  

I think we should specify whether the connection id should be the same?  I believe it should be, to ensure the routing is the same.

We should also specify what happens if the version negotiation packet includes the version you just tried to use.  I believe we should not try again.